dev/com.ibm.websphere.appserver.features/visibility/public/websocket-1.0/com.ibm.websphere.appserver.websocket-1.0.feature

```diff
@@ -15,8 +15,12 @@ Subsystem-Name: Java WebSocket 1.0
  io.openliberty.javaee.websocket-1.0, \
  com.ibm.websphere.appserver.eeCompatible-7.0; ibm.tolerates:="6.0,8.0"
 -bundles=com.ibm.ws.wsoc, \
- io.openliberty.wsoc.1.0.internal,\
- io.openliberty.wsoc.ssl.internal
+ io.openliberty.wsoc.1.0.internal, \
+ io.openliberty.wsoc.ssl.internal, \
+ io.openliberty.io.netty, \
+ io.openliberty.io.netty.ssl, \
+ io.openliberty.netty.internal, \
+ io.openliberty.netty.internal.impl
```

Comment:
It makes more sense for these to be in a private feature that this feature depends on so you aren't copying the same set of bundles into all of these features and can just rely on an`io.openliberty.netty.internal-1.0` feature or something like that.